README.md

```diff
@@ -23,115 +22,117 @@ $ yarn add webpack-sentry-plugin --dev
-
-  ```js
-  var config = {
-    plugins: [
-      new SentryPlugin({
-        releaseBody: function(version) {
-          return { 
-            version,
-            refs: [
-               repository: 'project-repo',
-               commit: process.env.GIT_SHA
-            ]
-          }
+- `releaseBody`: Object or function that returns the body that will be sent to Sentry. Defaults to sending the version and projects (which is sufficient to create a basic new release in Sentry).
+  - The function is given two arguments: `version` and `projects`. `version` is the result of the `release` object (string or function output); `projects` is the `project` configuration parameter (converted to an array if a single string is provided).
+  - The most common use case for overriding this field is Sentry's [release commits](https://docs.sentry.io/learn/releases/#releases-are-better-with-commits) feature. To use this, define `releaseBody` per the example below (providing the most recent commit hash through whatever means works best for your build setup). See the Sentry documentation for more details and options.
+
+```js
+var config = {
+  plugins: [
+    new SentryPlugin({
+      releaseBody: function(version, projects) {
+        return { 
+          version,
+          projects,
+          refs: [
+             repository: 'project-repo',
+             commit: process.env.GIT_SHA
+          ]
         }
-      })
-    ]
-  }
-  ```
+      }
+    })
+  ]
+}
+```
 
-- `suppressErrors`: Display warnings instead of failing webpack build
+- `suppressErrors`: Display warnings for any errors instead of failing webpack build
 
-- `suppressConflictError`: Similar to `suppressErrors`, but only supresses release conflict errors - useful in case webpack compilation is done during deploy on multiple instances
+- `suppressConflictError`: Similar to `suppressErrors`, but only supresses release conflict errors - useful in case webpack compilation is done during deploy on multiple instances. (Release conflict errors are HTTP 409 errors thrown by Sentry when the same source map file is uploaded to the same release multiple times.)
 
-- `baseSentryURL`: URL of Sentry instance. Shouldn't need to set if using sentry.io, but useful if self hosting
+- `baseSentryURL`: URL of Sentry instance. Unnecessary if using sentry.io, but useful if self-hosting. Should be a fully qualified domain name with the suffix `/api/v0`, i.e. `https://mysentryinstance.com/api/v0`
 
-- `deleteAfterCompile`: Boolean determining whether source maps should be deleted after the webpack compile finishes. Defaults to `false`
+- `deleteAfterCompile`: Boolean determining whether source maps should be deleted on the build server after the webpack compile finishes. Useful if you want to upload source maps to Sentry but do not want to make them publicly available to users of your application. Defaults to `false`
```

Comment:
Hm. Is this really what it's for? Simply having the source maps doesn't mean that they're publicly available.

Reply:
That's what I use it for haha. I didn't think too much about other possible use cases. I can just delete the sentence to avoid confusion though.